docs/Operation-Architecture/Lifecycle_Management/Object_Lifecycle_Management.rst

```diff
@@ -95,10 +87,8 @@ for more examples and explanations on lifecycle rules.
 Once the lifecycle rules on the bucket are set, the rules apply to all
 objects in the specified bucket.
 
-`Go back to Advanced Workflows`_
-
-`Go back to CRR Retry`_
-
-.. _`Go back to Advanced Workflows`: ../Orbit_UI/Advanced_Workflows/Advanced_Workflows.html
+.. toctree::
+  :maxdepth: 1
+  :hidden:
 
-.. _`Go back to CRR Retry`: ../Zenko_CLI/CRR_Retry.html
+  lifecycle_expiration_policies
```

Comment:
I think this line is not wanted

Reply:
No, it's there on purpose, though the logic/implementation is not excellent. When I get the toctree declarations better dialed in, this stuff will make better sense.